NEWS.md

```diff
@@ -2,6 +2,12 @@
 
 * `@family` lists are now ordered more carefully, "foo1" comes after "foo" (#1563, @krlmlr).
 
+* `@importFrom magrittr "%>%"` works again, as does
+  ``@importFrom rlang `:=` `` (#1570, @MichaelChirico). The form
```

Comment:
I think I'd start the bullet with something like `` `@importFrom` with quoted non-syntactic functions works again, e.g. ...`.

Reply:
Thanks for the feedback, PTAL.